src/appellate/appellate.js

```diff
@@ -124,18 +143,53 @@ AppellateDelegate.prototype.handleDocketDisplayPage = async function () {
   if (options['recap_enabled']) {
     let callback = (ok) => {
       if (ok) {
-        addAlertButtonInRecapAction(this.court, pacer_case_id);
+        addAlertButtonInRecapAction(this.court, this.pacer_case_id);
         history.replaceState({ uploaded: true }, '');
         this.notifier.showUpload('Docket uploaded to the public RECAP Archive.', function () {});
       }
     };
 
-    this.recap.uploadDocket(this.court, pacer_case_id, document.documentElement.innerHTML, 'APPELLATE_DOCKET', (ok) =>
-      callback(ok)
+    this.recap.uploadDocket(
+      this.court,
+      this.pacer_case_id,
+      document.documentElement.innerHTML,
+      'APPELLATE_DOCKET',
+      (ok) => callback(ok)
     );
-
-    this.attachRecapLinksToEligibleDocs();
   } else {
     console.info(`RECAP: Not uploading docket. RECAP is disabled.`);
   }
 };
+
+AppellateDelegate.prototype.handleAttachmentPage = async function () {
+  this.pacer_case_id = await APPELLATE.getCaseId(this.tabId, this.queryParameters, this.docId);
+
+  if (this.pacer_case_id) {
```

Comment:
Shouldn't this be:

```suggestion
  if (!this.pacer_case_id) {
```

Reply:
Yes! You're right. That line should check if the `pacer_case_id` is empty. I will change this. 

Thanks